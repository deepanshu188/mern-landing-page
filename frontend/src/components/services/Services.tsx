import Button from '../UI/Button';
import ServicesContainer from './ServicesContainer';
import app_development from '../../assets/img/services/app.svg';
import web_development from '../../assets/img/services/web.svg';
import graphic_designing from '../../assets/img/services/graphic.svg';
import digital_marketing from '../../assets/img/services/digital.svg';

export const Services = () => {
  return (
    <ServicesContainer>
      <div className="title">
        <h1 className="heading">Explore Our Services</h1>
        <p className="paragraph">
          We are self-service data analytics software that lets you
          create visually.
        </p>
      </div>
      <div className="cards">
        <div className="card">
          <img src={app_development} />
          <p className="heading">App Development</p>
          <p className="paragraph">
            ipsum dolor sit amet, consectetur adipiscing elit. Nullam
            euismod metus vel sem bibendum, a bibendum justo tempor.
            Sed vel lectus
          </p>
        </div>
        <div className="card">
          <img src={web_development} />
          <p className="heading">Web Designing</p>
          <p className="paragraph">
            ipsum dolor sit amet, consectetur adipiscing elit. Nullam
            euismod metus vel sem bibendum, a bibendum justo tempor.
            Sed vel lectus
          </p>
        </div>
        <div className="card">
          <img src={graphic_designing} />
          <p className="heading">Graphic Designing</p>
          <p className="paragraph">
            ipsum dolor sit amet, consectetur adipiscing elit. Nullam
            euismod metus vel sem bibendum, a bibendum justo tempor.
            Sed vel lectus
          </p>
        </div>
        <div className="card">
          <img src={digital_marketing} />
          <p className="heading">Digital Marketing</p>
          <p className="paragraph">
            ipsum dolor sit amet, consectetur adipiscing elit. Nullam
            euismod metus vel sem bibendum, a bibendum justo tempor.
            Sed vel lectus
          </p>
        </div>
      </div>
      <Button>Learn More</Button>
    </ServicesContainer>
  );
};
