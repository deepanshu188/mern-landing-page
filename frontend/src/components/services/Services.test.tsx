import { expect, test } from 'vitest';
import renderer from 'react-test-renderer';
import { Services } from './Services';

function toJson(component: renderer.ReactTestRenderer) {
  const result = component.toJSON();
  expect(result).toBeDefined();
  expect(result).not.toBeInstanceOf(Array);
  return result as renderer.ReactTestRendererJSON;
}

test('services', () => {
  const component = renderer.create(<Services />);
  const tree = toJson(component);
  expect(tree).toMatchSnapshot();
});
