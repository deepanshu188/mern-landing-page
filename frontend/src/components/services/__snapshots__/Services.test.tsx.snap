// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`services 1`] = `
<section
  className="sc-gsTDqH cYbAvo"
>
  <div
    className="title"
  >
    <h1
      className="heading"
    >
      Explore Our Services
    </h1>
    <p
      className="paragraph"
    >
      We are self-service data analytics software that lets you create visually.
    </p>
  </div>
  <div
    className="cards"
  >
    <div
      className="card"
    >
      <img
        src="/src/assets/img/services/app.svg"
      />
      <p
        className="heading"
      >
        App Development
      </p>
      <p
        className="paragraph"
      >
        ipsum dolor sit amet, consectetur adipiscing elit. Nullam euismod metus vel sem bibendum, a bibendum justo tempor. Sed vel lectus
      </p>
    </div>
    <div
      className="card"
    >
      <img
        src="/src/assets/img/services/web.svg"
      />
      <p
        className="heading"
      >
        Web Designing
      </p>
      <p
        className="paragraph"
      >
        ipsum dolor sit amet, consectetur adipiscing elit. Nullam euismod metus vel sem bibendum, a bibendum justo tempor. Sed vel lectus
      </p>
    </div>
    <div
      className="card"
    >
      <img
        src="/src/assets/img/services/graphic.svg"
      />
      <p
        className="heading"
      >
        Graphic Designing
      </p>
      <p
        className="paragraph"
      >
        ipsum dolor sit amet, consectetur adipiscing elit. Nullam euismod metus vel sem bibendum, a bibendum justo tempor. Sed vel lectus
      </p>
    </div>
    <div
      className="card"
    >
      <img
        src="/src/assets/img/services/digital.svg"
      />
      <p
        className="heading"
      >
        Digital Marketing
      </p>
      <p
        className="paragraph"
      >
        ipsum dolor sit amet, consectetur adipiscing elit. Nullam euismod metus vel sem bibendum, a bibendum justo tempor. Sed vel lectus
      </p>
    </div>
  </div>
  <button
    className="sc-bdfCDU eTpbqK"
  >
    Learn More
  </button>
</section>
`;
